DataDog/documentation
# App Sec OOTB Rules

### What does this PR do?
- Updates the `security_rules.py` file to pull in new app sec rules;
- Updates Cloud Configuration and Infrastructure Configuration to Posture Management (Cloud) and Posture Management (Infra) for product context
- Updates pull configs for the build
- Updates content on OOTB rules page for product clarity

### Motivation
Docs OKR / Jira

### Preview
https://docs-staging.datadoghq.com/sarina/app-sec-ootb-rules/security_platform/default_rules/

### QA steps

On the OOTB page:

1. Click all product-named buttons to ensure they work
2. Under `application security`, click on individual rules to make sure they are populating

---

### Reviewer checklist
